WordPress/gutenberg
# Fix: mis-matching aria-label issue in styles revision

## What?
- fix `aria-label` mismatching content text issue.

## Why?
fixes https://github.com/WordPress/gutenberg/issues/66525


## How?
- removed extra `aria-label` text from `revision-buttons` component.

## Testing Instructions
- create new style revision.
- open style revision history.
- check any users visible text and screen reader pronunciation text.

## Screenshots or screencast 


https://github.com/user-attachments/assets/8020c334-790c-495a-83a5-10d0ac4bbbe1
